src/common/accelcmn.cpp

```diff
@@ -112,6 +112,23 @@ static const struct wxKeyName
     { WXK_WINDOWS_RIGHT,    /*TRANSLATORS: Name of keyboard key*/ wxTRANSLATE("Windows_Right") },
     { WXK_WINDOWS_MENU,     /*TRANSLATORS: Name of keyboard key*/ wxTRANSLATE("Windows_Menu") },
     { WXK_COMMAND,          /*TRANSLATORS: Name of keyboard key*/ wxTRANSLATE("Command") },
+    { WXK_BROWSER_BACK,     /*TRANSLATORS: Name of keyboard key*/ wxTRANSLATE("Back") },
```

Comment:
I have never seen menu items with browser/media keys as accelerator either. I will remove it from the PR.